client/src/main/java/se/fortnox/reactivewizard/client/HttpClient.java

```diff
@@ -420,6 +421,10 @@ private void setHeaderParams(RequestBuilder request, Method method, Object[] arg
                 }
             }
         }
+        
+       if (isNullOrEmpty(request.getHeaders().get("Host"))) {
```

Comment:
The build is failing because indentation is wrong here and on line 427, just add one more space and you're good to go 👍